controller/rules/k8s.go

```diff
@@ -0,0 +1,182 @@
+package rules
+
+import (
+	"fmt"
+
+	kuberules "github.com/amalgam8/amalgam8/pkg/adapters/rules/kubernetes"
+	"github.com/amalgam8/amalgam8/pkg/api"
+	kubepkg "github.com/amalgam8/amalgam8/pkg/kubernetes"
+	"github.com/pborman/uuid"
+	kubeapi "k8s.io/client-go/pkg/api"
+	"k8s.io/client-go/pkg/api/unversioned"
+	"k8s.io/client-go/rest"
+)
+
+var (
+	// BulkNotSupportedError is returned when unsupported bulk operations are attempted.
+	BulkNotSupportedError = fmt.Errorf("bulk operations not supported")
+)
+
+// K8S controller. Currently no bulk insert/update operations are supported and
+// the controller is bound to a single namespace.
+type K8S struct {
+	// kubernetes REST client
```

Comment:
Doc comment not strictly required (since not exported). Not sure what "the style guide" says about whitespace... Ditto below